src/posix/platform/hdlc_interface.cpp

```diff
@@ -600,6 +600,15 @@ int HdlcInterface::OpenFile(const Url::Url &aRadioUrl)
         {
             tios.c_cflag |= CRTSCTS;
         }
+        else
+        {
+            tios.c_cflag &= ~(CRTSCTS);
+
+            // Deassert DTR and RTS
+            int flags;
+            flags = TIOCM_DTR | TIOCM_RTS;
+            VerifyOrExit((ioctl(fd, TIOCMBIC, &flags)) != -1, perror("tiocmbic"));
```

Comment:
I think we can remove one `( )`?
```suggestion
            VerifyOrExit(ioctl(fd, TIOCMBIC, &flags) != -1, perror("tiocmbic"));
```

Reply:
yes